relayer/relays/beacon/syncer/api.go

```diff
@@ -32,7 +33,8 @@ type BeaconClientTracker interface {
 }
 
 var (
-	ErrNotFound = errors.New("not found")
+	ErrNotFound                        = errors.New("not found")
+	ErrSyncCommitteeUpdateNotAvailable = errors.New("not found")
```

Comment:
The errors have the same message.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/661/commits/b6a353d0049de0d2fc379e28fd99c5c96a43ef37.